.go-version

```diff
@@ -1 +1 @@
-1.18.1
+1.18.5
```

Comment:
why not 1.19?

Reply:
wait for 1.19.1 ? I don't have a strong opinion but I'll prefer we pick .1 for every major version.

Reply:
Kubernetes doesn't wait 🤷‍♂️ 

All minor go releases are actually soaked in google prod before release, and we should be much less sensitive to go changes than Kubernetes ... we don't have 5k node scale issues or high performance requirements.

Reply:
Bumped.